database/migrations/2017_03_31_211222_create_feedback_tables.php

```diff
@@ -151,7 +151,7 @@ public function up()
               'type_id' => '4',
               'form_id' => '2',
               'slug' => 'professional4',
-              'question' => 'They were professional and well delivered',
+              'question' => 'They service they provided was professional and well delivered.',
```

Comment:
Not sure that this question would make sense for pilots, seeing as they don't provide a service